js_modules/dagster-ui/packages/ui-core/src/assets/Stale.tsx

```diff
@@ -66,49 +99,64 @@ export const StaleReasonsLabel = ({
   );
 };
 
-export const StaleReasonsTags = ({
-  liveData,
-  include,
+export const StaleReasonsTag = ({
   assetKey,
+  liveData,
+  include = 'all',
   onClick,
 }: {
   assetKey: AssetKeyInput;
-  include: 'all' | 'upstream' | 'self';
   liveData?: StaleDataForNode;
+  include?: 'all' | 'upstream' | 'self';
   onClick?: () => void;
 }) => {
-  if (!isAssetStale(liveData) || !liveData?.staleCauses.length) {
-    return null;
-  }
+  const staleTag = useMemo(() => {
+    if (!isAssetStale(liveData) || !liveData?.staleCauses?.length) {
+      return <div />;
+    }
+    const label = (
+      <Caption>Outdated ({numberFormatter.format(liveData.staleCauses.length)})</Caption>
+    );
+    return (
+      <Popover
+        content={
+          <StaleCausesPopoverSummary liveData={liveData} assetKey={assetKey} include={include} />
+        }
+        position="top"
+        interactionKind="hover"
+        className="chunk-popover-target"
+      >
+        <BaseTag
+          fillColor={Colors.backgroundYellow()}
+          textColor={Colors.textYellow()}
+          icon={<Icon name="changes_present" color={Colors.textYellow()} />}
+          label={
+            onClick ? (
+              <ButtonLink underline="never" onClick={onClick} color={Colors.textYellow()}>
+                {label}
+              </ButtonLink>
+            ) : (
+              label
+            )
+          }
+        />
+      </Popover>
+    );
+  }, [assetKey, include, liveData, onClick]);
+
+  const isNew = liveData?.changedReasons?.includes(ChangeReason.NEW);
 
   return (
-    <>
-      {Object.entries(groupedCauses(assetKey, include, liveData)).map(([label, causes]) => (
-        <Popover
-          key={label}
-          content={<StaleCausesPopoverSummary causes={causes} />}
-          position="top"
-          interactionKind="hover"
-          className="chunk-popover-target"
-        >
-          <BaseTag
-            fillColor={Colors.backgroundYellow()}
-            textColor={Colors.textYellow()}
-            interactive={!!onClick}
-            icon={<Icon name="changes_present" color={Colors.textYellow()} />}
-            label={
-              onClick ? (
-                <ButtonLink underline="never" onClick={onClick} color={Colors.textYellow()}>
-                  {label}
-                </ButtonLink>
-              ) : (
-                label
-              )
-            }
-          />
-        </Popover>
-      ))}
-    </>
+    <Box
+      flex={{gap: 4, alignItems: 'center', justifyContent: 'space-between'}}
+      padding={{horizontal: 4}}
+      style={{height: 24}}
+    >
+      {staleTag}
+      {isNew ? (
```

Comment:
Does this mean the change reason tag will only show if the asset is new? we'd also like to show for assets that aren't new but do have changes, the case when `changedReasons = [ChangeReason.INPUTS]` etc